major_event_log/feeds.py

```diff
@@ -10,23 +11,95 @@
 class MajorEventLogFeed(Atom1Feed):
     """Custom Atom Feed for Event objects.
 
-    The only change between this custom feed and the base Atom Feed
-    is that this feed specifies its mime_type as 'application/xml'.
+    The only changes between this custom feed and the base Atom Feed
+    are that this feed specifies its mime_type as 'application/xml',
+    pagination is added, the "alternate" link is removed.
     """
+
     mime_type = 'application/xml'
 
+    def add_root_elements(self, handler):
+        handler.addQuickElement(u"title", self.feed['title'])
+        handler.addQuickElement(
+            u"link", "", {u"rel": u"self", u"href": self.feed['feed_url']}
+        )
+        handler.addQuickElement(u"id", self.feed['id'])
+        handler.startElement(u"author", {})
+        handler.addQuickElement(u"name", self.feed['author_name'])
+        handler.addQuickElement(u"uri", self.feed['author_link'])
+        handler.endElement(u"author")
+        handler.addQuickElement(u"subtitle", self.feed['subtitle'])
+        handler.addQuickElement(
+            u"link",
+            "",
+            {u"rel": u"first", u"href": self.feed['link'] + '?p=1'}
+        )
+        handler.addQuickElement(
+            u"link",
+            "",
+            {
+                u"rel": u"last",
+                u"href": self.feed['link'] + '?p=%s' % self.feed['last_link']
```

Comment:
This method has a weird blend of string concatenation styles going on like in the line above. It would be good to update the code to be more consistent.